bibliopixel/drivers/serial/driver.py

```diff
@@ -13,7 +13,7 @@ class DriverSerial(DriverBase):
     def __init__(self, type, num, dev="",
                  c_order=ChannelOrder.RGB, SPISpeed=2,
                  gamma=None, restart_timeout=3,
-                 deviceID=None, hardwareID="1D50:60AB",
+                 device_id=None, hardwareID="1D50:60AB",
```

Comment:
renaming all these to be consistent... this failed because it was deviceID but you later use device_id, which didn't exist.